### PR TITLE
Add _id to vrs models

### LINKED ISSUE #175: Include all _id values from VRS/VRSATILE models

### PR DESCRIPTION
**Pull request by korikuzma**
Closes #175

### CODE REVIEW

File: variation/normalize.py [hunk 1]
```diff
@@ -57,7 +59,7 @@ def normalize(self, q: str, validations: ValidationSummary,
 
                 variation = valid_result.variation
 
-                variation_id = variation.pop('_id')
+                variation_id = variation['_id']
```
Comment (jsstevenson) [inline variation/normalize.py:62]: double quotes?

edit: looks like this file has only been partially switched to double quotes so ignore if that's a later task

File: variation/query.py [hunk 2]
```diff
@@ -114,7 +116,9 @@ def to_vrs(self, q: str)\
 
         if not translations:
             if q and q.strip():
-                translations = [Text(definition=q)]
+                text = models.Text(definition=q)
```
Comment (jsstevenson) [inline variation/query.py:119]: I don't know if this is an actionable comment here, but is it weird that we have Text classes in both vrs-python and vrsatile-pydantic?

**Review by jsstevenson — APPROVED** [review]

File: variation/normalize.py (hunk 1 above)
Comment (korikuzma) [reply variation/normalize.py:62]: Ya i plan on doing that in #185

File: variation/query.py (hunk 2 above)
Comment (korikuzma) [reply variation/query.py:119]: Oh it definitely is weird and it felt weird doing it. I needed to use vrs-python in order to create the digest, but in the response models we use the pydantic models. I think maybe I should create an issue to create the digests in vrsatile.pydantic?

**Pull request merged**